certified-connectors/Dokobit Portal/apiDefinition.swagger.json

```diff
@@ -126,6 +137,7 @@
                   "type": "array",
                   "items": {
                     "type": "object",
+                    "required": ["name", "content"],
```

Comment:
@epelanis These input fields "name", "content" have been changed from optional to required. can you please confirm that no existing flows or users will be affected by this change?

Reply:
These were always required on our end. However, it was not indicated as such in the documentation. Therefore, users would need to go through the trial and error process to discover which parameters are required. This is not a breaking change but an improvement to the clarity of the documentation.
Instead of getting data validation errors, users will see which parameters are mandatory.

Reply:
@epelanis thank you so much for the confirmation.